### ISSUE TITLE
write doesn't have all the keyword arguments

### ISSUE THREAD

**Issue by stassats** [drmeister/clasp#67]
At least :circle is missing. Same deal with write-to-string.

**Comment by Shinmera** [comment]
Can confirm on b05884f.
```
> (write "foo" :circle T)

Condition of type: SIMPLE-PROGRAM-ERROR
Unrecognized keyword argument error: :CIRCLE
```

**Comment by drmeister** [comment]
And WRITE-TO-STRING uses WRITE - so W-T-S works as well

**Comment by Shinmera** [comment]
fb10987 is not on testing yet, cannot confirm.

**Comment by Shinmera** [comment]
Fixed on 482d6ca5bd8ad30ff8c9b6a835989f3aec395e1a
```
> (write "foo"
       :array            *print-array*                   
       :base             *print-base*                    
       :case             *print-case*                    
       :circle           *print-circle*                  
       :escape           *print-escape*                  
       :gensym           *print-gensym*                  
       :length           *print-length*                  
       :level            *print-level*                   
       :lines            *print-lines*                   
       :miser-width      *print-miser-width*             
       :pprint-dispatch  *print-pprint-dispatch*         
       :pretty           *print-pretty*                  
       :radix            *print-radix*                   
       :readably         *print-readably*                
       :right-margin     *print-right-margin*
       :stream           *standard-output*)
"foo"
"foo"
> (write-to-string "foo"
       :array            *print-array*                   
       :base             *print-base*                    
       :case             *print-case*                    
       :circle           *print-circle*                  
       :escape           *print-escape*                  
       :gensym           *print-gensym*                  
       :length           *print-length*                  
       :level            *print-level*                   
       :lines            *print-lines*                   
       :miser-width      *print-miser-width*             
       :pprint-dispatch  *print-pprint-dispatch*         
       :pretty           *print-pretty*                  
       :radix            *print-radix*                   
       :readably         *print-readably*                
       :right-margin     *print-right-margin*
       :stream           *standard-output*)

"\"foo\""
```
